activerecord/lib/active_record/connection_adapters/postgresql/oid/json.rb

```diff
@@ -2,32 +2,7 @@ module ActiveRecord
   module ConnectionAdapters
```

Comment:
`OID::Json` is base class of `OID::Jsonb`.
https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/postgresql/oid/jsonb.rb#L5
Do I need to change to `class Jsonb < Type::Json`?